spec/migrations/20170317153953_copy_agent_id_to_miq_server_id_in_jobs_table_spec.rb

```diff
@@ -0,0 +1,22 @@
+require_migration
+
+describe CopyAgentIdToMiqServerIdInJobsTable do
+  let(:job_stub) { migration_stub(:Job) }
+  let(:job_name) { "Hello Test Job" }
+
+  migration_context :up do
+    it "copies data from 'agent_id' to 'miq_server_id' column on jobs table" do
+      job_stub.create!(:name => job_name, :agent_id => 111)
+      migrate
+      expect(Job.find_by(:name => "Hello Test Job").miq_server_id).to eq 111
```

Comment:
Missed a use of `job_name`:

`expect(Job.find_by(:name => job_name).miq_server_id).to eq 111`